src/js/outreach/views/opt-in_views.js

```diff
@@ -0,0 +1,153 @@
+import hbs from 'handlebars-inline-precompile';
+import { View } from 'marionette';
+
+import 'scss/modules/buttons.scss';
+import 'scss/modules/forms.scss';
+
+import './opt-in.scss';
+
+const OptInView = View.extend({
+  template: hbs`
+    <div class="opt-in__icon opt-in__icon--warn">{{fat "hand-wave"}}</div>
+    <h2 class="opt-in__heading-text">Hi, we need to confirm your contact info. Please enter your information below, so that we can share health resources with you.</h2>
+    <div class="opt-in__field">
+      <label class="opt-in__field-label">Your first name</label>
+      <input
+        type="text"
+        class="input-primary opt-in__field-input js-first-name"
+        placeholder="Enter your first name"
+        value="{{ firstName }}"
+      />
+    </div>
+    <div class="opt-in__field">
+      <label class="opt-in__field-label">Your last name</label>
+      <input
+        type="text"
+        class="input-primary opt-in__field-input js-last-name"
+        placeholder="Enter your last name"
+        value="{{ lastName }}"
+      />
+    </div>
+    <div class="opt-in__field">
+      <label class="opt-in__field-label">Your date of birth</label>
+      <input
+        type="text"
+        class="input-primary opt-in__field-input js-dob"
+        placeholder="Enter your date of birth"
+        value="{{ dob }}"
+      />
+    </div>
+    <h3 class="opt-in__heading-text u-margin--t-32 u-margin--b-16">How may we share health resources with you?</h3>
+    <div class="opt-in__field">
+      <label class="opt-in__field-label">Your mobile phone number</label>
+      <input
+        type="text"
+        class="input-primary opt-in__field-input js-phone"
+        placeholder="Enter mobile phone number"
+        value="{{ phone }}"
+      />
+    </div>
+    <div class="opt-in__field">
+      <label class="opt-in__field-label">Your email address</label>
+      <input
+        type="text"
+        class="input-primary opt-in__field-input js-email"
+        placeholder="Enter email address"
+        value="{{ email }}"
+      />
+    </div>
+    <p class="opt-in__disclaimer">By clicking Submit you agree to receive SMS text message notifications. You may opt out at any time.</p>
+    <button class="opt-in__submit button--green w-100 js-submit" disabled>Submit</button>
+  `,
+  ui: {
+    firstName: '.js-first-name',
+    lastName: '.js-last-name',
+    dob: '.js-dob',
+    phone: '.js-phone',
+    email: '.js-email',
+    submit: '.js-submit',
+  },
+  triggers: {
+    'keyup @ui.firstName': 'change:first:name',
+    'keyup @ui.lastName': 'change:last:name',
+    'keyup @ui.dob': 'change:dob',
+    'keyup @ui.phone': 'change:phone',
+    'keyup @ui.email': 'change:email',
```

Comment:
I switched to using `keyup` here (instead of `onChange`).

With `keyup`, we can update the submit button state (disabled vs. enabled) as the user is typing. With `onChange`, we'd have to wait until the user un-focused the input field.

Reply:
what about the `input` event?

Reply:
`input` is preferred, not sure why I originally used `keyup` instead.